identity-example/src/main/java/com/stripe/android/identity/example/MainActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.stripe.android.identity.example
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.stripe.android.identity.IdentityVerificationSheet
+import com.stripe.android.identity.example.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+
+    private val binding by lazy {
+        ActivityMainBinding.inflate(layoutInflater)
+    }
+
+    lateinit var identityVerificationSheet: IdentityVerificationSheet
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(binding.root)
+
+        identityVerificationSheet =
+            IdentityVerificationSheet.create(
+                this,
+                IdentityVerificationSheet.Configuration(
+                    merchantLogo = R.drawable.stripe_logo,
+                    stripePublishableKey = "pk_test"
+                )
+            )
+
+        binding.startVerification.setOnClickListener {
+            identityVerificationSheet.present(
+                verificationSessionId = "testVerificationSessionId",
+                ephemeralKeySecret = "testEphemeralKeySecret"
+            ) {
+                Log.d(TAG, "verification result: $it")
```

Comment:
nit: may be better to use a snackbar like we do in the payment sheet examples and I think maybe cardscan. [example you can copy paste if you decided to add one](https://github.com/stripe/stripe-android/blob/master/paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt#L124)

Reply:
sg! updated